spec/controllers/feedbacks_controller_spec.rb

```diff
@@ -79,6 +81,15 @@ def valid_session
     end
   end
 
+  describe "GET edit_additional" do
+    it "assigns the requested feedback as @feedback" do
+      feedback = Feedback.create! valid_attributes
+      get :edit_additional, {:id => feedback.to_param, :review_id => @review.id}, valid_session
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.